index.js

```diff
@@ -5,14 +5,48 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = 'purple'
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const firstProfImg = document.querySelector('#profile img')
+firstProfImg.src = "pics/raffy.jpg"
+firstProfImg.alt = "Raffy"
 
+const firstProfName = document.querySelector('div#profile h2')
+firstProfName.innerHTML = "Raffy"
+
+const firstProfNickname = document.querySelector('div#profile em')
+firstProfNickname.innerHTML = "Rafferty"
```

Comment:
If you're just changing the text of an element, you should use `textContent` or `innerText` here instead. Ultimately they'll do the same thing, but looking at [this resource](https://stackoverflow.com/questions/35213147/difference-between-textcontent-vs-innertext), which do you think would be better?